skyportal/handlers/api/source.py

```diff
@@ -1298,6 +1336,45 @@ class Validator(Schema):
             query_results["sources"] = obj_list
 
         query_results = recursive_to_dict(query_results)
+        if includeGeojson:
+            features = []
+
+            # currently necessary for the sources to render
+            features.append(
```

Comment:
What's up with these three identical `Feature` instances?

Reply:
For some reason, I had to put a few down to get them to render on the front-end... Might just me being dumb though, IDK.

Reply:
Can't we just add the relevant data to the demo data?

Reply:
I am not quite sure what you mean. The geojson feature table is generated from the sources list.

Reply:
I'm not sure I understand why this is needed, but it definitely doesn't seem like it should be necessary

Reply:
I couldn't agree more. I am pretty sure this is just me not being good at front end stuff. But if I include it, points do appear ;)

Reply:
We need to figure out what's going on here, why these dummy points are needed, and either get rid of them or at least document the valid reason for them being here. And instead of all this repeated code, please do something more concise like `features.extend([Feature(...)] * 3)`